src/App/reducers/post.js

```diff
@@ -0,0 +1,27 @@
+import { message } from "antd";
+import { ADD_POST, POST_ERROR } from "../actions/types";
+
+const initialState = {
+  content: null
+};
+
+export default function(state = initialState, action) {
+  const { type, payload } = action;
+
+  switch (type) {
+    case ADD_POST:
+      message.success("Posted successfully");
+      return {
+        ...state,
+        ...payload
+      };
+    case POST_ERROR:
+      message.error("Error");
+      return {
+        ...payload
```

Comment:
You need to spread the state there, cause state is immutable and our previous data will be lost if you don't do so.